content/sifixes/src/items/scroll.uc

```diff
@@ -17,22 +17,44 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+/*  2016-07-07 List updated to display items in the order the party finds them,
+ *  instead of the original haphazard list. Order is Avatar, Shamino, Dupre, Iolo.
+ *  Also fixes the spacing so adding companion-authors does not push text down.
+ *
+ *  2016-07-15 Added Quality 122 section to allow accusing Marsten of betraying
+ *  his city, and if Pomdirgun is dead, sets the flag Monitor NPCs check.
+ */
+
 void Scroll shape#(0x31D) ()
 {
 	var quality = get_item_quality();
 	UI_play_sound_effect2(0x5E, item);
 	book_mode();
-	if (quality == 0xBD)
+	// Found in the Goblin King's treasure room, signed by Marsten, Lord of Monitor.
+	if (quality == 0x7A)
+	{   // Marsten can be accused.
+		gflags[CAN_ACCUSE_MARSTEN] = true; 
+		// If the Goblin King is dead.
+		if (UI_get_item_flag(GOBLIN_KING, DEAD))
+		{
+			gflags[POMDIRGUN_IS_DEAD] = true; // Conversations in Monitor will reflect his death.
+		}
+	}
+
+	if (quality == 0xBD) // Equipment Scroll
 	{
 
-		say("The list of items which we found ourselves with after the storm:~~");
+		say("The list of items which we found ourselves with after the storm:~");
 		say("Prepared by Shamino.");
 		if (gflags[DUPRE_MADE_EQUIPMENT_LIST])
 			say("With additional notes by Dupre.");
+        else say (""); // added a blank line
 		if (gflags[IOLO_MADE_EQUIPMENT_LIST])
 			say("And further comments by Iolo, since being freed from that vile Monitorian prison cell!");
-		say("~~");
+        else say ("~~"); // Iolo's comment gets to three lines.
```

Comment:
Indentation is wrong here, the else should be on the same level as the if and the line below should be indented as well.